docs/xks/operator-guide/github.md

```diff
@@ -63,6 +63,14 @@ jobs:
       AZURE_CREDENTIALS_PROD: ${{ secrets.AZURE_CREDENTIALS_PROD }}
 ```
 
+### Self-hosted runners
+
+GitHub workflows is still rather immature and have a number of features missing.
+One of these are that it's currently not possible to use self-hosted runners in github organization X while calling on workflows in github organization Y.
```

Comment:
```suggestion
It's currently not possible to use self-hosted runners in github organization X while calling on workflows in github organization Y.
```
I think mentioning the maturity of GH workflows isn't relevant here, even if I share and understand your frustration.

If you disagree, here's my more conservative suggestion with just grammar fix:
```suggestion
GitHub workflows is still rather immature and a number of features are missing from it.
One of these is that it's currently not possible to use self-hosted runners hosted in GitHub organization X while calling on workflows in GitHub organization Y.
```